python/ray/_private/worker.py

```diff
@@ -2620,10 +2643,57 @@ def get(
         return values
 
 
+@PublicAPI
+def _write_channel(value: Any, object_ref: ObjectRef, num_readers: int):
+    worker = global_worker
+    worker.check_connected()
+
+    if num_readers <= 0:
+        raise ValueError("``num_readers`` must be a positive integer.")
+
+    try:
+        serialized_value = worker.get_serialization_context().serialize(value)
+    except TypeError as e:
+        sio = io.StringIO()
+        ray.util.inspect_serializability(value, print_file=sio)
+        msg = (
+            "Could not serialize the put value " f"{repr(value)}:\n" f"{sio.getvalue()}"
+        )
+        raise TypeError(msg) from e
+
+    worker.core_worker.put_serialized_object_to_mutable_plasma_object(
+        serialized_value,
+        object_ref,
+        num_readers,
+    )
+
+
+@PublicAPI
+def _create_channel(
+    buffer_size: int,
```

Comment:
can we make it kwargs?

Reply:
Feel like that's easier to undertsand

Reply:
Not sure I see the point of doing that (python always lets you call with kwargs-style).